File: Python/Project Euler #020.py
# Solución para el problema #20 de Project Euler
# "Encuentra la suma de los dígitos del número 100!"
# https://projecteuler.net/problem=20

def sum_digits(n):
    s = 0 #Variable suma
    while n:
        s += n % 10 #Se suman los digitos, uno por uno
        n //= 10
    return (s)

respuesta = 1
x = 1
while x <= 100:
    respuesta *= x
    x += 1
    print (respuesta)
  
print ("La respuesta es: ") #La respuesta es 648
print (sum_digits(respuesta))

File: Visual Basic .NET/Project Euler #020.vb
Module Module1

    Sub Main()
        Dim n As Integer = 1
        Dim n1 As Integer = 0
        Dim respuesta As Long = 1

        ' Solución para el problema #20 de Project Euler
        ' "Encuentra la suma de los dígitos del número 100!"
        ' https://projecteuler.net/problem=20
        ' Código espagueti de 2016; optimizar

        Do Until n = 100

            'Overflow cuando n = 95 o n = 14, dependiendo de a partir de donde se multiplique

            'n1 = n
            'n += 1
            'respuesta = respuesta * n * n1 '.ToString?
            'Console.Write(n)
            'Console.Write("   ")
            'Console.Write(n1)

            respuesta = (2 * 3) * (4 * 5) * 6 * 7 * 8 * 9 * 11 * 12 * 13

            'Se pueden remover los ceros de las decenas; por ejemplo, 10, 20, 30, multiplicando por 1, 2 y 3 respectivamente

            Console.Write("   ")
            Console.WriteLine(respuesta)

        Loop

        Console.WriteLine(respuesta)
        Console.ReadLine()

    End Sub
    
 End Module
